agent/structs/aclfilter/filter.go

```diff
@@ -258,7 +258,9 @@ func (f *Filter) filterServiceNodes(nodes *structs.ServiceNodes) bool {
 			continue
 		}
 		removed = true
-		f.logger.Debug("dropping node from result due to ACLs", "node", structs.NodeNameString(node.Node, &node.EnterpriseMeta))
+		f.logger.Debug("dropping service node from result due to ACLs",
+			"node", structs.NodeNameString(node.Node, &node.EnterpriseMeta),
+			"service", structs.ServiceIDString(node.ServiceID, &node.EnterpriseMeta))
```

Comment:
~ I noticed a few other fns below log `"service"` as `svc.CompoundServiceID()`. Is there a reason for the difference here, or should we align those as well?

Reply:
Good call. I just updated this 👍